src/documents.h

```diff
@@ -22,13 +22,19 @@
 
 #include <gio/gio.h>
 
+typedef enum {
+  DOCUMENT_FLAG_NONE      = 0,
+  DOCUMENT_FLAG_FOR_SAVE  = (1 << 1),
+  DOCUMENT_FLAG_WRITABLE  = (1 << 2),
+  DOCUMENT_FLAG_DIRECTORY = (1 << 3),
+  DOCUMENT_FLAG_DELETABLE = (1 << 4),
```

Comment:
Sorry for the persistence here, but now that NONE = 0, the first entry should start with `1 << 0`, and the rest should be updated

Reply:
No worries, I am not very familiar with C "best practices" yet.